calcite/README.md

```diff
@@ -107,4 +107,4 @@ For further details see [Calcite Web TypeFaces](https://esri.github.io/calcite-w
 
 ## Demo app
 
-In the [demo](./demo/) folder, there is a Qt demo application project (calcite_test.pro) that showcases many of the Qt toolkit Calcite components being used.
\ No newline at end of file
+In the [demo](./demo/) folder, there is a Qt demo application project (calcite_test.pro) that showcases many of the Qt toolkit Calcite components being used.
```

Comment:
I am not sure why this change shows up. I didn't change anything here. I only changed the typo above.